lib/pages/chat/events/message.dart

```diff
@@ -579,6 +576,9 @@ class Message extends StatelessWidget {
     );
   }
 
+  bool displayUsername(bool ownMessage) =>
+      ownMessage || event.room.isDirectChat || !isSameSender(nextEvent, event);
```

Comment:
no need to display name in `direct chat`

Reply:
methods name was not good, it should be "hideDisplayName"